fulp_modules/features/toys/code/toys.dm

```diff
@@ -24,3 +24,12 @@
 	attack_verb_continuous = list("dust")
 	attack_verb_simple = list("dust")
 	squeak_override = list('sound/effects/supermatter.ogg'= 1)
+
+/obj/item/toy/plush/pico
+	name = "Pico plushie"
+	desc = "A plushie of an alarmed middle schooler with orange hair and a green T-Shirt. Maybe somethng's happening at his school."
```

Comment:
"somethng"

Reply:
Hm? Is somethng wrong with that line?

Jokes aside all of these should be fixed with the new push